### ISSUE TITLE
v3.0.7.6 version with "Accept terms of use"

### ISSUE THREAD

**Issue by Laurenzak** [exstreme/Jcomments-Recaptcha2#32]
I am working on a new version from yours v3.0.7.5 version.
What's new?
1.- I have added in the backend a new option.  This option allows to show or hide a checkbox in the JComments form: "Accept Terms of Use". I have also updated the language files. I have also added this option in the sql file to create this option in the database.
2.- I have slightly modified the .css file to make the form responsive.
3.- I have updated redirections from "index.php" to homepage.
4.- I have modified "About JComments" to show this web and let people know what version they have installed.
I have called this version v3.0.7.6 and now I am programming and finishing the installer.
How can I contact you and help?

**Comment by exstreme** [comment]
Hello!
[Here](https://joomlaforum.ru/index.php?action=profile;u=192862) can you find some contacts

**Comment by Laurenzak** [comment]
Hi! exstreme. The Joomla! forum: joomlaforum.ru, does not allow me to send you private messages. I've attached the JComments v3.0.7.6 file if you want to test it and upload it to your JComments project. I will explain all the modifications I have made from your version 3.0.7.5:
1.- I have slightly modified the file "/components/com_jcomments/tpl/default/style.css" to make the form responsive.
2.- I have modified the component redirections from "index.php" to the home page. File: "components/com_jcomments/jcomments.php" line 127 and 130
3.- I added two options in the backend to show/hide an "accept terms of use" checkbox in the JComments form in the frontend. Option in "Layout" tab and "Permission" tab. Files affected:

- Backend languages: "administrator/language/en-GB/en-GB.com_jcomments.ini" and "administrator/language/ru-RU/ru-RU.com_jcomments.ini". Added AP_ENABLE_ACCEPT_TERMS_OF_USE, A_ABOUT_JCOMMENTS_GITHUB_PROJECT and AP_ENABLE_ACCEPT_TERMS_OF_USE_DESC tags.
- Fronted languages: All language files (43 files "xx-XX.com_jcomments.ini") have been updated. Added FORM_ACCEPT_TERMS_OF_USE and ERROR_CHECKBOX_TERMS_OF_USE_NO_SELECTED tags.
- Backend option: file "administrator/components/com_jcomments/models/forms/settings.xml" line 400 and permissions.xml line 40.
- Database options (2 options): file "administrator/components/com_jcomments/install/sql/default.settings.sql". The last two lines.
- Code to show checkbox. File "components/com_jcomments/tpl/default/tpl_form.php". LIne 128
- Code to create a variable: File "components/com_jcomments/jcomments.php" line 400
- Code to check if the checkbox has been activated in the backend and if the user has checked the checkbox. File "components/com_jcomments/jcomments.ajax.php" line 210 and saveComment function.

4.- Fixed error in recaptcha v2. Deleted RECAPTCHA item in JComments CAPTCHA options. ERROR fixed: disable recaptcha v2 from Joomla plugins and then select RECAPTCHA option in JComments.
5.- Added in "About JComments" JComments version and link to github.

Tested on PHP 7.4, Joomla 3.9.23/3.9.24
Regards!
[com_jcomments_v3.0.7.6.zip](https://github.com/exstreme/Jcomments-Recaptcha2/files/5838903/com_jcomments_v3.0.7.6.zip)








![001](https://user-images.githubusercontent.com/77511019/105100281-60836d00-5aad-11eb-9ba9-e1853976197c.png)
![002](https://user-images.githubusercontent.com/77511019/105100286-611c0380-5aad-11eb-8486-82853f4a6f72.png)
